pkg/sql/jobs/jobs.go

```diff
@@ -726,7 +726,10 @@ func RunAndWaitForTerminalState(
 		var status Status
 		var jobErr gosql.NullString
 		var fractionCompleted float64
-		err := sqlDB.QueryRow(`SELECT status, error, fraction_completed FROM [SHOW JOBS] WHERE id = $1`, jobID).Scan(
+		err := sqlDB.QueryRow(
+			`SELECT status, error, fraction_completed
+         FROM crdb_internal.jobs
+        WHERE id = $1`, jobID).Scan(
```

Comment:
Since this doesn't use any internal columns, I'd prefer to leave it as `SHOW JOBS` so we actually exercise the `SHOW JOBS` code path somewhere.

Reply:
Also, I'm not sure I understand your indentation scheme.